packages/batcher/batcher-transaction-poster/src/evm.ts

```diff
@@ -66,6 +75,14 @@ class EvmBatchedTransactionPoster extends BatchedTransactionPosterBase {
     const serializedSignedTx = ethers.Transaction.from(
       await this.provider.getConnection().api.signTransaction(populatedTx)
     );
+    return { txRequest, populatedTx, serializedSignedTx };
+  };
+
+  public override postMessage = async (
+    msg: string,
+    hashes: string[]
+  ): Promise<[number, string]> => {
+    const { txRequest, serializedSignedTx } = await this.buildTransaction(msg);
```

Comment:
`this.provider.finalizeTransaction` actually makes network calls, so it's best not to repeat this work if possible